pyrefly/lib/alt/call.rs

```diff
@@ -633,6 +634,47 @@ impl<'a, Ans: LookupAnswer> AnswersSolver<'a, Ans> {
                 };
                 self.construct_class(cls, args, keywords, range, errors, context, hint)
             }
+            CallTarget::SelfClass(cls) => {
+                if cls.has_qname("typing", "Any") {
```

Comment:
maybe this should be factor out so we can share it between ClassTarget::Class and ClassTarget::SelfClass since this code is identical